testing/README.md

```diff
@@ -2,26 +2,27 @@
 
 ## Philosophy
 * Test code is a first-class part of the deliverable, not an optional nice-to-have.
-* When scopeing / estimating work: always factor in testing. 
-* Untestable code = bad code.
```

Comment:
I'm hesitant to include such a strong statement suggesting that untestable code is "bad code". Some of the reasons are as follows:
1. This guide will be referenced by contractors that help us on client projects in the future. The possibility of a contractor developer feeling accused of writing "bad code" isn't a very friendly welcome to the team.
2. Even once we get into a consistent rhythm writing tests, we will find ourselves in situations where we have code that can't or won't be tested. Again, if we are suggesting this is "bad code", I now feel like a "bad" developer. I'd rather we have a positive spin as much as we can. 🙂 

I couldn't think of a different way to word the original statement so my suggestion ended up removing the line. If we still want to keep this line but reword it, that's great! I'm open to suggestions. 😉

Reply:
I think there might be some confusion between untest**able** vs untest**ed** code. Code that has no tests may or may not be bad. But "untestable code" is code that _cannot_  be tested, and I don't want to sugar-coat that fact that this is a serious problem and not up to our standards. The same factors that make code untestable make it brittle, confusing, and prone to bugs. I think we all agree those things are "bad". I agree, it's a strong statement, but also true. In this context, it feels appropriate to optimize for clarity and directness rather than the hypothetical sensitivities of a contractor. Having said that, it's certainly not my goal to upset anyone. So maybe this feels more clear and/or comfortable: 

> Testability is an attribute of good code. Code that cannot be tested is a problem.

I think it's important for us to have a clear stance on this, so I don't want to simply remove it.

Did you mean to delete the preceding bullet point, too?  Including testing in scoping/estimating seems pretty uncontroversial, so I suspect that might be a typo. But let me know.

Reply:
Thanks for the feedback and extra context, @emersonthis! You make some great points. 

> Testability is an attribute of good code. Code that cannot be tested is a problem.

This works for me! 🎉 

> Did you mean to delete the preceding bullet point, too? Including testing in scoping/estimating seems pretty uncontroversial, so I suspect that might be a typo. But let me know.

Great catch! Unfortunately, the diff isn't as clear within the context of this thread. My change was only fixing a typo ("scopeing" => "scoping"). 😉

Reply:
@emersonthis I've added your suggested verbiage. 👍 

https://github.com/cloudfour/guides/pull/68/commits/3f0a63b6a21eb38a144ed5e90133f7e1a95906dc